### PR TITLE
Add footer

### LINKED ISSUE #14: Add Footer
Description
As a user, I want to see a good Footer, with informations about the company.

Task
Choose a Footer template on TailwindUI page and add to the application.

### PR DESCRIPTION
**Pull request by PabloFLPs**
Just adding a chosen TailwindUI Footer template. Closes #14.

### CODE REVIEW

**Review by Erickinhou — COMMENTED** [review]
I think this is  a commit from another pr. Keep changes that isn't on the master separated from other pr's.

image

File: src/components/Footer.tsx [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+        name: 'GitHub',
+        href: '#',
+        icon: (props:any) => (
+        <svg fill="currentColor" viewBox="0 0 24 24" {...props}>
+            <path
+            fillRule="evenodd"
+            d="M12 2C6.477 2 2 6.484 2 12.017c0 4.425 2.865 8.18 6.839 9.504.5.092.682-.217.682-.483 0-.237-.008-.868-.013-1.703-2.782.605-3.369-1.343-3.369-1.343-.454-1.158-1.11-1.466-1.11-1.466-.908-.62.069-.608.069-.608 1.003.07 1.531 1.032 1.531 1.032.892 1.53 2.341 1.088 2.91.832.092-.647.35-1.088.636-1.338-2.22-.253-4.555-1.113-4.555-4.951 0-1.093.39-1.988 1.029-2.688-.103-.253-.446-1.272.098-2.65 0 0 .84-.27 2.75 1.026A9.564 9.564 0 0112 6.844c.85.004 1.705.115 2.504.337 1.909-1.296 2.747-1.027 2.747-1.027.546 1.379.202 2.398.1 2.651.64.7 1.028 1.595 1.028 2.688 0 3.848-2.339 4.695-4.566 4.943.359.309.678.92.678 1.855 0 1.338-.012 2.419-.012 2.747 0 .268.18.58.688.482A10.019 10.019 0 0022 12.017C22 6.484 17.522 2 12 2z"
+            clipRule="evenodd"
+            />
+        </svg>
```
Comment (Erickinhou) [inline src/components/Footer.tsx:48]: I think is better to keep the svg on separeted files. In this case you can create (for example) a file called facebook SVG and import on the file.

File: src/components/NavBar.tsx [hunk 2]
```diff
@@ -64,74 +64,8 @@ export default function NavBar() {
...
-                            className={classNames(active ? 'bg-gray-100' : '', 'block px-4 py-2 text-sm text-gray-700')}
-                          >
-                            Sign out
-                          </a>
-                        )}
-                      </Menu.Item>
-                    </Menu.Items>
-                  </Transition>
-                </Menu>
-              </div>
```
Comment (Erickinhou) [inline src/components/NavBar.tsx:131]: This is from another pr.

**Review by Erickinhou — APPROVED** [review]
It's working! Just some adjustments.

File: src/components/NavBar.tsx (hunk 2 above)
Comment (PabloFLPs) [reply src/components/NavBar.tsx:131]: Oh, my apologies.. I possibly created this branch from the branch that I was doing this commit. :(

**Pull request merged**